lib/mondrian/olap/connection.rb

```diff
@@ -7,7 +7,8 @@ def self.create(params)
         connection
       end
 
-      attr_reader :raw_connection, :raw_catalog, :raw_schema, :raw_schema_reader
+      attr_reader :raw_connection, :raw_catalog, :raw_schema,
+                  :raw_schema_reader, :cache_control
```

Comment:
It would be better to use `raw_cache_control` to distinguish naming of "raw" Mondrian cache control object from our own CacheControl class instance.